server/.env.example

```diff
@@ -0,0 +1,9 @@
+DATABASE_HOST="YOUR MYSQL DATABASE HOST"
+DATABASE_PORT="YOUR MYSQL DATABASE PORT"
+DATABASE_USER="YOUR MYSQL DATABASE USER"
+DATABASE_PASSWORD="YOUR MYSQL DATABASE PASSWORD"
+DATABASE_NAME="YOUR MYSQL DATABASE NAME"
+EMAIL_SENDER="SENDER'S EMAIL"
+EMAIL_PASSWORD="SENDER'S EMAIL PASSWORD"
+EMAIL_ME2="RECIEVER EMAIL"
```

Comment:
Instead of "RECIEVER EMAIL" make it "SAMPLE EMAIL1" and "SAMPLE EMAIL2"

Reply:
Updated the required changes.. Kindly review @Adi-ty